### ISSUE TITLE
Preselecting item from Editable-Select when bound to a Model.

### ISSUE THREAD

**Issue by HollandRisley** [vitalets/angular-xeditable#100]
I can't get the Angular-xeditable Editable-Select to Select the current item when it is bound to a nested JSON object.  Data all saves fine, but it just doesn't show up the currently selected item in the Select box, which is driving me nuts, as I am sure I am missing something obvious. I have created a JSFiddle here:

http://jsfiddle.net/NfPcH/1031/

This is the code:

```
<span editable-select="data.organisation.OrganisationType"
              e-ng-options="type.Name for type in data.types">
            {{data.organisation.OrganisationType.Name}}
        </span>
```

These are the 2 data objects:

```
$scope.data.organisation =  {
"Id":1,
"Name":"My Organisation",
"OrganisationType":{"Id":2,"Name":"Internal"}
 } 

$scope.data.types = [
{"Id":1,"Name":"Client"},
{"Id":2,"Name":"Internal"},
{"Id":3,"Name":"Cold"}
] 
```

It works fine when I bind it to the ID within the Nested Object, but then it only changes the Id in my nested object, and I then have to manually filter and change the Name part of the object, which works, but I am sure there must be a better way.

**Comment by tarlepp** [comment]
@rmariuzzo please do, I'm having the same problem. Also issue #238 is linked to this.

**Comment by adaro** [comment]
@HollandRisley - any chance I can see that directive you built? I am having the same problem :P

**Comment by ckosloski** [comment]
@eugef I think this can be closed since #238 was closed.
